endpoint/activate.go

```diff
@@ -3,12 +3,17 @@ package endpoint
 import (
 	"context"
 
+	"github.com/pkg/errors"
+
+	"github.com/Scalingo/go-utils/logger"
 	"github.com/Scalingo/sand/api/params"
 	"github.com/Scalingo/sand/api/types"
-	"github.com/pkg/errors"
 )
 
 func (r *repository) Activate(ctx context.Context, n types.Network, endpoint types.Endpoint, params params.EndpointActivate) (types.Endpoint, error) {
+	log := logger.Get(ctx)
```

Comment:
Don't you want to add network and endpoint ID to the logger? Same question on the other file you modified.

Reply:
It's the entity loading the data which adds it in the logger, so the HTTP handlers or worker